src/components/PaginationButton/index.tsx

```diff
@@ -0,0 +1,20 @@
+import styles from "./PaginationButton.module.scss";
+import { IPaginationButtonProps } from "./types";
+
+const PaginationButton: React.FC<IPaginationButtonProps> = ({
+  label,
+  isChecked,
+}) => {
+  return (
+    <button
+      className={[
+        styles["pagination-button"],
+        isChecked && styles["pagination-button_checked"],
+      ].join(" ")}
```

Comment:
Please make a utility function for it. Follow this approach when copying/pasting the code a lot. Or at least remove it from JSX by creating a const in your component